tests/integration/security/authorization_test.go

```diff
@@ -314,73 +308,77 @@ func TestAuthorization_Deny(t *testing.T) {
 			if t.Clusters().IsMulticluster() {
 				t.Skip()
 			}
+			for _, srcCluster := range t.Clusters() {
+				ns := apps.Namespace1
+				rootns := newRootNS(t)
+				b := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
+				c := apps.C.Match(echo.Namespace(apps.Namespace1.Name()))
+				vm := apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))
+				for _, dst := range [][]echo.Instances{{b, c}, {b, vm}, {vm, c}} {
```

Comment:
this (`{b, c}, {b, vm}, {vm, c}`) is really confusing, to make the test workload agnostic, we should always test one target workload at a time.

The proper way to change this test is to split the current test to 2 rounds:
1. for the old `policy-b-deny`, test it against b and vm
2. for the old `policy-c-deny`, test it against b and vm
You then just loop over the {b, vm} once.

Reply:
Thanks for the comments. Removed the combinations of targets and the iterations.